config/locales/es.yml

```diff
@@ -2537,7 +2537,7 @@ es:
               </li>
             </ul>
             <p>
-              Encuentre un desglose más detallado de los créditos y deducciones no admitidos este año aquí.
+              Encuentre un desglose más detallado de los créditos y deducciones no admitidos este año <a href="https://www.fileyourstatetaxes.org/en/md/faq/md_not_supported_credits_and_subtractions" rel="noopener nofollow" target="_blank">aquí</a>.
```

Comment:
provide the /es link? `https://www.fileyourstatetaxes.org/es/md/faq/md_not_supported_credits_and_subtractions` [pebble]